monolith/src/main/java/com/society/server/dto/payload/SignupDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.society.server.dto.payload;
+
+import com.society.server.utils.validators.EmailValidator;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
```

Comment:
Don't use lombok all and use specific imports.

Reply:
I don't think that's important, intellij does it automatically when imports are >=5